list_participants_tsv_levels.py

```diff
@@ -139,16 +145,11 @@ def list_levels(
 
     Adds any undefined level not found in the data dictionary.
     """
-    levels = {}
-    if (
-        participants_dict
-        and participants_dict.get(column)
-        and participants_dict[column].get("Levels")
-    ):
-        levels = participants_dict[column]["Levels"]
+    levels = get_levels_from_data_dict(participants_dict, column)
+    if levels:
         output = append_levels(output, levels, column, row_template)
 
-    actual_levels = list(participants[column].unique())
+    actual_levels = [str(x) for x in participants[column].unique()]
```

Comment:
just out of curiosity: what's the reason for this change?
It'll work well for us, I think, because we cast everything to string. Just wanted to make sure I understand

Reply:
ah nvm, I read the above description in the PR. Missed this.